js/jquery.terminal-src.js

```diff
@@ -1919,7 +1918,7 @@
     var format_last_re = /\[\[[!gbiuso]*;[^;]*;[^\]]*\]?$/i;
     var format_exec_re = /(\[\[(?:[^\]]|\\\])*\]\])/;
     $.terminal = {
-        version: '{{VER}}',
+        version: '0.11.17',
```

Comment:
Here should be '{{VER}}' because it's source file and it's replaced by version by make.